relay-general/src/store/normalize/contexts.rs

```diff
@@ -76,13 +76,32 @@ fn normalize_runtime_context(runtime: &mut RuntimeContext) {
     }
 }
 
+/// Parses the Windows build number from the description and maps it to a marketing name.
+/// Source: https://en.wikipedia.org/wiki/List_of_Microsoft_Windows_versions
+/// Note: We cannot distinguish between Windows Server and PC versions, so we map to the PC versions
```

Comment:
Open question: What percentage of SDKs that send `raw_description` run on Windows Server? For those, this PR might actually make things worse.